examples/bri-3/src/bri/zeroKnowledgeProof/agents/ccsmAnchorHashLocalStorage.agent.ts

```diff
@@ -0,0 +1,92 @@
+import { Mapper } from '@automapper/core';
+import { InjectMapper } from '@automapper/nestjs';
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { PrismaService } from '../../../../prisma/prisma.service';
+import { CCSMAnchorHash } from '../models/ccsmAnchorHash';
+import { Document } from '../models/document';
+import { CCSM_ANCHOR_HASH_NOT_FOUND_ERR_MESSAGE } from '..//api/err.messages';
+
+@Injectable()
+export class CCSMAnchorHashLocalStorageAgent extends PrismaService {
```

Comment:
convention is that storageAgent refers to local storage for all other modules. I would continue using the same convention here and would rename this  CCSMAnchorHashLocalStorageAgent to CCSMAnchorHashStorageAgent. The other one, which is specific becuase it stores on chain, should have a specific unconventional name such as CCSMAnchorHashCcsmStorageAgent ( look at my other comment for ditching ccsm everywhere else, then this name makes more sense ;))

Reply:
I agree with this and would prefer to ditch ccsm. Using ccsmStorageAgent sounds better.

Reply:
@Therecanbeonlyone1969 Can we rename `CCSMAnchorHash` to simply `AnchorHash` and name the blockchain storage agent as `ccsmStorageAgent`?

Reply:
My proposal would be the following:

anchorHashStogateAgent - storing into local postgres db
anchorHashCcsmStorageAgent - storing on chain

I would treat the ccsm acronym as every other noun when it comes to casing.

Reply:
And +1 on @biscuitdey proposal. Let's ditch ccsm in front of anchor hash completely. It just creates visual noise. @Therecanbeonlyone1969